README.md

```diff
@@ -1,8 +1,207 @@
+By default, the service runs on 8998 port.
+
+#### Using service manager 2
+
+The following sm2 command starts the service but its also part of `GMS_ALL` profile
+
+`sm2 --start GOODS_MOVEMENT_SYSTEM_PORT_API`
+
+the logs can be viewed by running `sm2 --logs GOODS_MOVEMENT_SYSTEM_PORT_API` in a different terminal window
+
+#### Using sbt
+For local development use `sbt run` but if its already running in sm2, we should stop it first before running sbt command
+
+The endpoint in this API is application-restricted.
+
+All ports requiring access to this API must be pre-authorized by HMRC.
+
+service logs can be viewed by running `sm --logs GOODS_MOVEMENT_SYSTEM_PORT_API`
+
+```GET  /:portId/arrivals/controlled```
+
+Port can pull all inspection required GMR's that are arriving at their port.
+
+Response Body Example
+```
+HTTP status: 200 (OK)
+
+[ {
+  "gmrId" : "GMRA00003I3M",
+  "isUnaccompanied" : false,
+  "vehicleRegNum" : "AB12 CDE",
+  "actualCrossing" : {
+    "routeId" : "15",
+    "localDateTimeOfArrival" : "2021-08-11T10:58"
+  },
+  "reportToLocations": [
+      {
+        "inspectionTypeId": "1",
+        "locationIds": [
+          "L0029A"
+        ]
+      }
+    ],
+} ]
+```
+
+```GET /:portId/departures/controlled```
+
+Port can pull all inspection required GMR's that are departing from their port.
+
+##### Response codes
+
+```
+HTTP status: 200 (OK)
+
+[ {
+  "gmrId" : "GMRA00003I3M",
+  "isUnaccompanied" : false,
+  "vehicleRegNum" : "AB12 CDE",
+  "actualCrossing" : {
+    "routeId" : "15",
+    "localDateTimeOfArrival" : "2021-08-11T10:58"
+  },
+  "reportToLocations": [
+      {
+        "inspectionTypeId": "1",
+        "locationIds": [
+          "L0029A"
+        ]
+      }
+    ],
+} ]
+```
+
+```GET /:portId/departures```
+
+Port can pull all GMR's that are departing from their port and view the ready to embark flag. 
+
+##### Response
+
+```
+HTTP status: 200 (OK)
+
+[ {
+  "gmrId" : "GMRD00003I3M",
+  "isUnaccompanied" : true,
+  "updatedDateTime" : "2021-08-31T08:52:04.791Z",
+  "inspectionRequired" : true,
+  "reportToLocations": [
+      {
+        "inspectionTypeId": "1",
+        "locationIds": [
+          "L0029A"
+        ]
+      }
+    ],
+  "trailerRegistrationNums" : [ "AB12 CFR", "GB18 WRD" ],
+  "containerReferenceNums" : [ "ABXY2", "GB456" ],
+  "checkedInCrossing" : {
+    "routeId" : "9",
+    "localDateTimeOfArrival" : "2021-08-11T10:58"
+  }
+},
+  {
+  "gmrId" : "GMRD00003I4M",
+  "isUnaccompanied" : true,
+  "updatedDateTime" : "2021-08-31T08:52:04.791Z",
+  "inspectionRequired" : false,
+  "trailerRegistrationNums" : [ "AB12 CFR", "GB18 WRD" ],
+  "containerReferenceNums" : [ "ABXY2", "GB456" ],
+  "readyToEmbark": true,
+  "checkedInCrossing" : {
+    "routeId" : "9",
+    "localDateTimeOfArrival" : "2021-08-11T10:58"
+  }
+} ]
+```
 
-This is a placeholder README.md for a new repository
+```GET  /reference-data ``` <br/>
+The Carrier can get a reference data at anytime.
```

Comment:
Can we change this: 

`The Carrier can get a reference data at anytime.`

to: 

`Retrieves reference data on request`